pyproject.toml

```diff
@@ -45,16 +45,16 @@ dev = [
     "coveralls",
     "pytest-mock",
     "pytest-cov",
-    "pygments==2.15.1",
-    "docutils==0.19",
+    "pygments==2.16.1",
+    "docutils==0.20.1",
     "furo",
     "pyright==1.1.322",
     "pyyaml==6.0",
-    "sphinx==7.1.2",
+    "sphinx==7.2.2",
     "sphinx-autobuild==2021.3.14",
     "sphinx-copybutton==0.5.2",
     # Intentionally kept at 2.3 until this bugfix is published: https://github.com/jdillard/sphinx-sitemap/pull/62
-    "sphinx-sitemap==2.3.0",
+    "sphinx-sitemap==2.5.1",
```

Comment:
I know they said the fix is published, but have we verified ourselves that incremental rebuilds are actually working on the new version?

Reply:
I have not verified the fix works for us.

Reply:
If running `python ./make.py html` twice in a row, the second time should be way faster, that's how we know it's working.  The bug affects its ability to un-pickle the cache from the previous invocation.